ios/FluentUI/Tab Bar/SideTabBar.swift

```diff
@@ -0,0 +1,304 @@
+        contain(view: backgroundView)
+
+        borderLine.translatesAutoresizingMaskIntoConstraints = false
+        addSubview(borderLine)
+
+        addSubview(topStackView)
+        addSubview(bottomStackView)
+
+        if #available(iOS 13, *) {
+            addInteraction(UILargeContentViewerInteraction())
+        }
+
+        accessibilityTraits = .tabBar
+
+        NSLayoutConstraint.activate([widthAnchor.constraint(equalToConstant: Constants.viewWidth),
+                                     borderLine.leadingAnchor.constraint(equalTo: trailingAnchor),
+                                     borderLine.bottomAnchor.constraint(equalTo: bottomAnchor),
+                                     borderLine.topAnchor.constraint(equalTo: topAnchor)])
+    }
+
+    private func setupLayoutConstraints() {
+        if layoutConstraints.count > 0 {
+            NSLayoutConstraint.deactivate(layoutConstraints)
+            layoutConstraints.removeAll()
+        }
+
+        if let avatarView = self.avatarView {
+            layoutConstraints.append(contentsOf: [
+                avatarView.topAnchor.constraint(equalTo: safeAreaLayoutGuide.topAnchor, constant: Constants.avatarViewTopPadding),
+                avatarView.centerXAnchor.constraint(equalTo: centerXAnchor),
+                avatarView.widthAnchor.constraint(equalToConstant: Constants.avatarViewSize),
+                avatarView.heightAnchor.constraint(equalToConstant: Constants.avatarViewSize),
+                topStackView.topAnchor.constraint(equalTo:avatarView.bottomAnchor , constant: Constants.avatarViewTopStackViewPadding)
+            ])
+        } else {
+            layoutConstraints.append(topStackView.topAnchor.constraint(equalTo: safeAreaLayoutGuide.topAnchor, constant: Constants.topStackViewTopPadding))
+        }
+
+        layoutConstraints.append(contentsOf: [
+            topStackView.widthAnchor.constraint(equalTo: widthAnchor),
+            topStackView.heightAnchor.constraint(equalToConstant: stackViewHeight(topStackView)),
+            bottomStackView.widthAnchor.constraint(equalTo: widthAnchor),
+            bottomStackView.heightAnchor.constraint(equalToConstant: stackViewHeight(bottomStackView)),
+            bottomStackView.bottomAnchor.constraint(equalTo: safeAreaLayoutGuide.bottomAnchor, constant: -Constants.bottomStackViewBottomPadding)
+        ])
+
+        NSLayoutConstraint.activate(layoutConstraints)
+    }
+
+    private func willSetItems(in section: Section) {
+        for subview in stackView(in: section).arrangedSubviews {
+            subview.removeFromSuperview()
+        }
+    }
+
+    private func didSetItems(in section: Section) {
+        let allItems = items(in: section)
+        let numberOfItems = allItems.count
+        if numberOfItems > Constants.maxTabCount {
+            preconditionFailure("tab bar items can't be more than \(Constants.maxTabCount)")
+        }
+
+        let stackView = self.stackView(in: section)
```

Comment:
>self [](start = 24, length = 4)

not needed

Reply:
It is needed, otherwise the compiler complains because the variable name and method name are conflicting. I wanted to keep the var name "stackView" so decided to add the explicit self.